test/fn/upload-docs.spec.js

```diff
@@ -72,7 +72,9 @@ describe('upload-docs', function() {
       .onCall(0).throws({ error: 'timeout' })
       .returns(Promise.resolve([{}]));
     fs.recurseFiles = () => new Array(10).fill('').map((x, i) => `${i}.doc.json`);
-    sinon.useFakeTimers(0);
+    const clock = sinon.useFakeTimers(0);
+    uploadDocs = rewire('../../src/fn/upload-docs');
+    uploadDocs.__set__('userPrompt', userPrompt);
```

Comment:
If we don't redefine `uploadDocs` here, the global `Date` object doesn't get updated with sinon's fake timer... sometimes I wish we wouldn't use `rewire` at all

Reply:
💀 Yeah, rewire is def a code smell.  If you need it in your tests, your code is not modular enough. This is janky but effective!  👍